packages/block-library/src/query/variations.js

```diff
@@ -14,6 +15,27 @@ import {
 } from './icons';
 
 const variations = [
+	{
+		name: '',
+		title: __( 'Posts List' ),
```

Comment:
Should it have a custom description?

Reply:
It could have and it would be only used in the Placeholder of Query, related to @mapk 's comment below.. If it wasn't used for Placeholder, I don't think it would be needed though. I'll have to check the Block Variations Picker probably

Reply:
It shows up in the preview:
<img width="655" alt="Screen Shot 2020-11-17 at 10 30 00" src="https://user-images.githubusercontent.com/699132/99372148-1371f780-28c0-11eb-9253-93226e095bd8.png">

Reply:
Good catch! I added it - thanks!